packages/arb-util/common/address.go

```diff
@@ -33,6 +33,22 @@ func NewAddressFromEth(a ethcommon.Address) Address {
 	return Address(a)
 }
 
+func AddressArrayFromEth(addresses []ethcommon.Address) []Address {
+	ret := make([]Address, 0, len(addresses))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [avoid magic numbers like '0', create a named constant for it](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)